src/NetTopologySuite/NetTopologySuite.csproj

```diff
@@ -31,7 +31,6 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(EnableApiCompat)' == 'true' ">
-    <PackageReference Include="Microsoft.DotNet.ApiCompat" Version="6.0.0-beta.21159.11" PrivateAssets="All" />
```

Comment:
I see a potential problem here, because `EnableApiCompat == True` but the reference to the package was removed.
Honestly I don't even know how this package is used, but I think it should be restored

Reply:
I wrote an article on my issue here https://github.com/NetTopologySuite/NetTopologySuite/issues/577
I didn't know how to fix it. It was suggested I remove a package and recompile. That seemed to work.

Reply:
My doubt is about pushing the package removal, I suggest to re-add the reference and then investigate the problem in #577

Reply:
This has to go away. If it does not work in your environment, you have to fix your environment.
If you want to turn it off temporarily, change `EnableApiCompat` to false (see 1st PropertyGroup)

Reply:
Yes but I don't know how to go about fixing my environment. I'm not even sure where to start. I did a google search and not much came up. I'm just unfamiliar with ApiCompat and how it works.

Reply:
> we have a fix for this

Not sure about that one, since the NuGet.config file does actually exist in this project... the question is what's causing it to get ignored?